### ISSUE TITLE
payroll & attendance api

### ISSUE THREAD

**Issue by mahesr** [sapplica/sentrifugo#341]
Team,

Are you having payroll & Attendance system or API. So we can do rest. Expecting Payroll & Attendance in this great system

**Comment by okechukwu0127** [comment]
yaaah... still expecting..

**Comment by OrbisTerrarumProLiberi** [comment]
its been 3 years. I think its dead

**Comment by mahesr** [comment]
Yes, it is look like they have closed the roadmap for further development. However they provide the required information & permission then others will do and can share into community. Hope they will consider this
